FordLabs/peoplemover
# Drag and drop refactor

## Issue
Resolves N/A

## What was done
- [x] Updated drag and drop to use [react-beautiful-dnd](https://github.com/atlassian/react-beautiful-dnd) (the same library being used in retroquest) and to not rely on redux
- [x] Delete all references of redux  

## How to test
1. Ensure drag and drop still works.
2. Ensure no functionality has changed in the app.
